### PR TITLE
Save pf-maint patch results

### PR DESCRIPTION
**Pull request by dwlfrth**
# Description
Saves the pf-maint patch result

# Impacts
addons/pf-maint

# Delete branch after merge
YES

### CODE REVIEW

**Comment by dwlfrth** [comment]
Review me.
Please take note of the use of backtick rather than "system". Will it break under certain circumstances ?

File: addons/pf-maint.pl [hunk 1]
```diff
@@ -146,7 +146,8 @@ sub apply_patch {
     my ( $data, $base, $head ) = @_;
     my $file = make_patch_filename( $base, $head );
     chdir $PF_DIR or die "cannot change directory $PF_DIR\n";
-    system "$PATCH_BIN -b -p1 < $file";
+    my @patch_result = `$PATCH_BIN -b -p1 < $file`;
```
Comment (jrouzierinverse) [inline addons/pf-maint.pl:149]: my $patch_result = `$PATCH_BIN -b -p1 < $file`;

File: addons/pf-maint.pl [hunk 2]
```diff
@@ -146,7 +146,8 @@ sub apply_patch {
...
+    write_file( catfile( $PATCHES_DIR, "${base}-${head}.result" ), @patch_result );
```
Comment (jrouzierinverse) [inline addons/pf-maint.pl:150]: write_file( catfile( $PATCHES_DIR, "${base}-${head}.result" ), $patch_result );

Comment (jrouzierinverse) [reply addons/pf-maint.pl:150]: No need to call the backtick in array context

**Review by jrouzierinverse — DISMISSED** [review]

**Comment by dwlfrth** [comment]
Changed... not quite sure it works tho...

**Comment by dwlfrth** [comment]
Review me

**Comment by fdurand** [comment]
Looks that it hide the patch result and also if we have to answer what to do on conflict.

**Comment by jrouzierinverse** [comment]
Please rebase

**Comment by dwlfrth** [comment]
I removed the milestone since i do not have time to work on this for v7.

**Comment by fdurand** [comment]
pf-maint doesn´t exist anymore

**Pull request closed without merging**